js/stringProcessing/subheadingsMatch.js

```diff
@@ -0,0 +1,31 @@
+var stringToRegex = require( "../stringProcessing/stringToRegex.js" );
+var replaceString = require( "../stringProcessing/replaceString.js" );
+var removalWords = require( "../config/removalWords.js" );
+
+/**
+ * Matches the keyword in an array of strings
+ *
+ * @param {Array} matches The array with the matched headings.
+ * @param {String} keyword The keyword to match
+ * @returns integer foundInHeader The number of occurrences of the keyword in the header.
```

Comment:
This should be `@returns {number} The number of occurrences of the keyword in the headings`